Update Snowflake and Databricks setup instructions

ajhorst opened the pull request:
Update Snowflake and Databricks setup instructions to reflect tweaks to mutation setup process;

markzegarelli reviewed (approved):
Thanks @ajhorst.

Can you remove the `.idea` folder from this PR before you merge please?

ajhorst commented:
> Thanks @ajhorst.
> 
> Can you remove the `.idea` folder from this PR before you merge please?

Looks like I'm the first one to edit the docs from a Jetbrains editor. I removed those files, and added the `.idea` folder to the gitignore file
